### ISSUE TITLE
Could you provide more guidance on how to do data preprocessing and network training？

### ISSUE THREAD

**Issue by Yellowshuohahaha** [rrgao/Point_Cloud_Colorization#1]
Thanks a lot!

**Comment by rrgao** [comment]
As for the data preprocessing:
 1)we extract semantic feature form pre trained semantic model and geo feature form pre trained point cloud matching model.
2)together with sparse coordinates of input points, we put it into our model.
3) for model training. We separately estimate the l and ab values. By using proposed constraints and loss functions. We follow the gan training model to train ours from scratch.

And, I will add the detailed source code of 1)preprocessing and 2)all pre-trained models in two weeks. 

Thanks for your question. Please keep an eye on it. 

发自我的iPhone

**Comment by Yellowshuohahaha** [comment]
That will be great, looking forward to it!

**Comment by Yellowshuohahaha** [comment]
@rrgao Hi, do you have time to add the pre-trained models and some guidances? Thank you very much!

**Comment by yogkul2000** [comment]
@rrgao can we get access to the pre-trained models and instructions how to replicate the paper results?
